### PR TITLE
Spelling param tests

### PR DESCRIPTION
**Pull request by reedstrm**
It was difficult to pass a false value for boolean options: had to send an empty string. This now accepts any of "False" "false" "0" "None" ""

Add test that does not correct spelling

### CODE REVIEW

File: tests/test_validate.py [hunk 1]
```diff
@@ -102,35 +102,6 @@ def test_non_words(client):
...
-        "tag_numeric": False,
-        "uid": None,
-        "uid_found": False,
-        "valid": False,
-    }
-
-    result = resp.json
-    assert result["computation_time"] != 0
-    expected["computation_time"] = result["computation_time"]
-    assert result == expected
```
Comment (LogisticAggression010) [inline tests/test_validate.py:131]: Just making sure that I understand this -- we are making sure that given actual meets expected, but the one thing that will be noisy is the computation time.  To get around this, you just set the expected to actual so that the test will pass?

**Review by LogisticAggression010 — APPROVED** [review]

File: tests/test_validate.py (hunk 1 above)
Comment (reedstrm) [reply tests/test_validate.py:131]: right. That's also threshhold test to make sure the time computation did something (it's not zero) but I figure the other parts of the result would probably reflect something drastic going wrong.
